substrate/frame/elections-phragmen/src/lib.rs

```diff
@@ -516,8 +577,14 @@ pub mod pallet {
 							.ok_or(Error::<T>::InvalidRenouncing)?;
 						// can't fail anymore.
 						let SeatHolder { deposit, .. } = runners_up.remove(index);
-						let _remainder = T::Currency::unreserve(&who, deposit);
-						debug_assert!(_remainder.is_zero());
+
+						let released = T::Currency::release_all(
+							&HoldReason::ReleaseForRunnerUp.into(),
```

Comment:
```suggestion
							&HoldReason::Candidacy.into()
```

Reply:
Since `Candidancy` is altered [here](https://github.com/paritytech/polkadot-sdk/pull/2011/files#r1639812844) to `Phragmen` - shouldn't `Phragmen` should be the HoldReason here?